kuksa_apps/s3/config.ini

```diff
@@ -1,19 +1,38 @@
-[kuksa_val]
+[general]
+# server type:
+# switch between kuksa_databroker and kuksa_val_server
+# default kuksa_val_server
+server_type = kuksa_val_server
+
+[kuksa_val_server]
+# kuksa_val_server IP address or host name
+ip = localhost
+port = 8090
+protocol = ws
+insecure = False
+# JWT security token file
+# token=../../kuksa_certificates/jwt/super-admin.json.token
+
+[kuksa_databroker]
+# kuksa_databroker IP address or host name
+ip = 127.0.0.1
+port = 55555
 
 [parquet]
 interval = 1
 max_num_rows = 10
-# seperated with ,
-path = Vehicle.Speed, Vehicle.Cabin.Infotainment.Navigation.DestinationSet.Longitude, Vehicle.Cabin.Infotainment.Navigation.DestinationSet.Latitude
-# FIXME path = Vehicle.Speed, Vehicle.Cabin.Infotainment.Navigation.DestinationSet
+# separated with ,
+paths = Vehicle.Speed, Vehicle.Cabin.Infotainment.Navigation.DestinationSet.Longitude, Vehicle.Cabin.Infotainment.Navigation.DestinationSet.Latitude
+# FIXME paths = Vehicle.Speed, Vehicle.Cabin.Infotainment.Navigation.DestinationSet
```

Comment:
This line is useless?

Reply:
This line is related to https://github.com/eclipse/kuksa.val/issues/132.

Reply:
Ah I see, I suggest just removing the  "#FIXME " line to keep it simple.